shuup/campaigns/models/campaigns.py

```diff
@@ -397,3 +397,6 @@ def save(self, **kwargs):
 
     def __str__(self):
         return self.code
+
+    def __unicode__(self):
```

Comment:
Don't define `__unicode__`, use the `@python_2_unicode_compatible` decorator on the class.